src/main/java/edu/harvard/iq/dataverse/util/FileUtil.java

```diff
@@ -192,11 +192,17 @@ public static String replaceExtension(String originalName) {
     }   
     
     public static String replaceExtension(String originalName, String newExtension) {
```

Comment:
I understand the urge to make the code more robust, but the changes done to replaceExtension quite possibly have downstream changes. Code may be expecting an error output on null or "" that will no longer be there. Please revert this change.

Reply:
Hi @matthew-a-dunlap, I understand that. I reverted the changes to the production class in 
3f20267 and a97ed76 and also commented out the test cases that would fail. If you want me to, I will uncomment these test cases and adjust them so that they pass with the current implementation.

Reply:
Yes feel free to get those tests to pass with how things are implemented currently.

Reply:
I adjusted the test cases where it was possible (f9d2001)